OpenRA.Mods.Common/Widgets/ConfirmationDialogs.cs

```diff
@@ -32,10 +36,12 @@ public static class ConfirmationDialogs
 			var cancelButton = prompt.GetOrNull<ButtonWidget>("CANCEL_BUTTON");
 			var otherButton = prompt.GetOrNull<ButtonWidget>("OTHER_BUTTON");
 
-			prompt.Get<LabelWidget>("PROMPT_TITLE").GetText = () => title;
+			var titleMessage = modData.Translation.GetString(title, titleArguments);
+			prompt.Get<LabelWidget>("PROMPT_TITLE").GetText = () => titleMessage;
 
 			var headerTemplate = prompt.Get<LabelWidget>("PROMPT_TEXT");
-			var headerLines = text.Replace("\\n", "\n").Split('\n');
+			var textMessage = modData.Translation.GetString(text, textArguments);
+			var headerLines = textMessage.Replace("\\n", "\n").Split('\n');
```

Comment:
Is the `Replace("\\n", "\n")` still needed with Fluent?

Reply:
It doesn't look like it, but the code below isn't functional.